add more details to the book README

bpires opened the pull request:
- [x] Add about the book
- [x] Add about the author
- [x] Add about the sponsor
- [x] Include a list of participants
- [x] Include visual aids
- [x] Create a banner
- [x] Keep it light-hearted and fun

Fixes #5

schalkneethling commented on working-the-command-line/README.md:
```diff
@@ -1,5 +1,81 @@
+
+
+
+<br>
```
Do we _need_ these `br` tags?

bpires replied:
Fair point!

schalkneethling commented on working-the-command-line/README.md:
```diff
@@ -1,5 +1,81 @@
+
+## 📙 About the book
+
+<div display="inline-block">
+<img align="left" width="125em" src="https://i.pinimg.com/736x/be/19/3f/be193ff8995c39cecbf58c7eeebbc3a9.jpg">
+</div>  
```
I reckon we can download these images and add them directly to the repository. It feels a little safer. For one thing, we can be sure the image will always render. The other thing is, someone might switch up the image on us :) Also, if we do not _need_ the HTML, I generally prefer to keep the Markdown to just Markdown. It allows us to, for example, add a linter without having to mess too much with formatting rules and exceptions.

bpires replied:
Thanks for pointing this out! It feels safer, indeed. And on that note, I think it's best to reduce the number of images to the minimum. I should have given this more thought 😅
In relation to HTML, I couldn't find a way to reduce the image and align it without recurring to the <img> tag. Please let me know if I'm missing something

schalkneethling commented on working-the-command-line/README.md:
```diff
@@ -1,5 +1,81 @@
+
+## 👤 About the author
+
+<div display="inline-block">
+<img align="left" width="100em" src="https://cdn.shopify.com/s/files/1/0051/7692/t/5/assets/author-remy-sharp-1608310970747.jpg">
+</div>  
+
```
Same here as my previous comment

schalkneethling commented on working-the-command-line/README.md:
```diff
@@ -1,5 +1,81 @@
+
+Remy tends to attract bugs and he enjoys squishing them. He uses the terminal daily.
+
+<br/>
```
Ditto :)

schalkneethling commented on working-the-command-line/README.md:
```diff
@@ -1,5 +1,81 @@
+
+## 🌠 About the sponsor
+
+<div display="inline-block">
+<img align="left" width="80em" src="https://media-exp1.licdn.com/dms/image/C560BAQGBqOQVMOSczw/company-logo_200_200/0/1519896899735?e=2147483647&v=beta&t=UcXA2wEcZhr_on_2GhnE4LOLB4jYPCsbexIcRFYjYXU">
+</div>  
```
Ditto

schalkneethling commented on working-the-command-line/README.md:
```diff
@@ -1,5 +1,81 @@
+
+
+|   |   |   |  
+|:---:|:---:|:---:|
+|<img width="85em" src="https://cdn.discordapp.com/avatars/748782811481243659/3a820d91a661a32184f027ede66ab52a.png?size=1024">&nbsp;|<img width="85em" src="https://cdn.discordapp.com/avatars/690807885617430558/d425f4cc87577004cd2888701d23d291.png?size=1024">|<img width="85em" src="https://cdn.discordapp.com/avatars/595880225846329345/c300fc8794de8266290aa1858a8d742e.png?size=1024">|
+|Zoe Sullivan (s/t)#0041|rafaelbpires#5725|Shriya Singh#9418|
+|<img width="85em" src="https://cdn.discordapp.com/avatars/199232269406175242/d7464363dc90d7abf8ec47a26e2496c5.png?size=1024">|<img width="85em" src="https://cdn.discordapp.com/avatars/800239929484771349/e94d2422b42da9d6d82a60398bd4a11a.png?size=1024">|
+|Hoodini#7495|MortimerShore#0708|
+
```
Again here, perhaps better to add the images directly to the repo.

schalkneethling reviewed (changes requested):
Thank you so much for taking the time to put this together @bpires 🎉 A couple of notes and suggestions.

bpires commented:
Thanks for the review @schalkneethling 🍄 I did some changes and remodelled the README based on your suggestions. I also included the latest changes in a new section. Please let me know what you think about it!

schalkneethling commented:
@bpires Do you have the source image you used for the header? Also, a link to the person’s Etsy store. Thanks!

bpires commented:
@schalkneethling it's https://www.etsy.com/shop/AshElizabethArt
And the image https://i.etsystatic.com/15228211/r/il/368ea9/2967627473/il_794xN.2967627473_tgil.jpg

schalkneethling commented:
@bpires what about something like this for the header template:
https://design.penpot.app/#/view/c74f9c80-3778-11ed-a5c2-cf2300df1f4f?page-id=c74f9c81-3778-11ed-a5c2-cf2300df1f4f&section=interactions&index=0&share-id=08802d90-3923-11ed-a5c2-cf2300df1f4f

bpires commented:
Loved the header @schalkneethling! Just two things:
1. There seems to be an issue with the font weight on the "network" word of the TMN logo
2. You may consider including the author of the book (like "Working the Command Line, by Remy Sharp").

Anyway, LGTM! :)

schalkneethling commented:
> [Something like this](https://www.canva.com/design/DAFM0MvHEBI/O9BnAMFD0aG0EW-fSv1cNA/edit?utm_content=DAFM0MvHEBI&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton) (Don't know how to use penpot yet, just a draft on Canvas to illustrate).

I really like that! I will update the one on PenPot.

schalkneethling commented:
@bpires This looks great! We just need to update the header graphic. Thanks!

bpires commented:
@schalkneethling, I've updated the banner and added links to the individual notes and resources of each chapter. I think we're good to go.

PS: The existent content regarding chapter 4 will be moved to the appropriate markdown file under chapter4 folder.

schalkneethling reviewed (changes requested):
Just two more items, almost there 🙌 - Thanks so much, @bpires, I cannot wait for this one to land.

bpires commented on working-the-command-line/README.md:
```diff
@@ -1,23 +1,62 @@
-# Working the command line
+![Bookclub banner](./assets/bookclub-banner.png)
```
Good to know, thanks!

schalkneethling commented on working-the-command-line/README.md:
```diff
@@ -1,23 +1,62 @@
-# Working the command line
+![Bookclub banner](assets/bookclub-banner.png)
```
@bpires I think these will work once we merge the PR. IF not, we can figure out why it does not then.

bpires commented:
I've accepted the suggestions and resized the image, leaving all in markdown with no HTML. 🎉 

PS: I'm curious: how do you preview the file output? Because it rendered fine for me here on GitHub with the `./` at the beginning of the file path. I'm asking just to be sure, I guess either way works.

schalkneethling commented:
> PS: I'm curious: how do you preview the file output? Because it rendered fine for me here on GitHub with the `./` at the beginning of the file path. I'm asking just to be sure, I guess either way works.

I was using "view rich diff", but it seems the "view file" option is the better one.

schalkneethling reviewed (approved):
Thank you for persisting with this one @bpires 🙌
